### ISSUE TITLE
Document Templates of different Sales Channels overwrite each other

### ISSUE THREAD

**Issue by AlexBachmann** [shopware/shopware#1590]
**Description**  

Currently, you can customize the document templates of your store by adding the templates (or template extensions) to the `src/Resources/views/documents` directory of your theme plugin. This all works well and good. However, once you create your second sales channel, which has it's own theme and document template, you'll create a new theme plugin, which has its own `src/Resources/views/documents` directory, containing the document templates for the second sales channel. Unfortunately, now both document templates are applied when creating the document. Each document template extending the other. 

**Possible Solution**  
A solution should not prevent plugins from extending document templates. However, Theme plugins, that is plugins that are declared as a theme, should only apply to those documents, whose corresponding orders are from the same sales channel as the corresponding theme.

In other words: If we have more than one theme installed, the template stack should only include the templates of those theme that is assigned to the sales channel that the order is from.

**Comment by AlexBachmann** [comment]
Any update on this? It seems to me to be a little bit excessive to create NEW document types for each new sales channel in order to create unique document templates for each and every one of them. This should be so much easier.

**Comment by MarcelSchmaeing** [comment]
Hello,

Within our company roadmap and work capacity, we try to address each bug or improvement request but admit that not each one will be resolved. To continue our culture of honesty and openness, we are closing this issue to focus on our roadmap on behalf of all Shopware users.

If you feel like this issue is still impacting you, please create a new issue and let us know.
Thank you again for your valuable feedback.
